### ISSUE TITLE
Using bootstrap for styling and layout

### ISSUE THREAD

**Issue by macroking** [SeleniumHQ/docs#160]

**Comment by eabedrapo** [comment]
Hi @shs96c 

When I opened the site in my mobile I realized the design isn't responsive. I think it's because the html is missing the viewport meta tag (I can add it quickly, so not big deal!).

In other hand I saw the beginning of a media block in the se.css and I see the breakpoint when playing with the browser width. But if Bootstrap is still in the table, I can help with that.

Otherwise I can move on and find something else to help with.

By the way I installed both Hugo and Jekyll in my computer but I don't know what to do next... so if it's OK I can just work with the HTML and CSS files already stored in this repo and start sending pull requests from my fork.

If you think it's easier if we talk about all of this, just DM me in Slack to coordinate how and when (given our time zone difference).

Have a good one!

PS: Of course I signed the Contributor License Agreement, but disclosing it just in case 😉
